Arashayo/TabletopHaven
# Remake game details

## Related Issue
https://app.clickup.com/t/86bx5b3f3
https://app.clickup.com/t/86bx5b284

## Changes list
- Remade game details view to work with all games
- Remade game details view to use database fields instead of calling API
- Updated ratings field to be a list of INTs
- Implemented random rating system on game initialization (2 to 15 INTs between 1 and 10)
- Added max and min players to database
- Slightly restyled the game details view
- Updated logo button to redirect to main page <WOW>
- made some changes in app.vue to correctly process right content

## Testing instructions

1. Run `node backend/src/createTables.js`
2. Run `node backend/src/fillGames.js` (feel free to Ctrl + C after few games, there aren't many needed for testing)
3. Optionally run `node backend/src/showGames.js`, review the ratings
4. Run both backend and frontend
5. Go to [localhost:3000/game/1](localhost:3000/game/1) or  [localhost:8081/game/1](localhost:8081/game/1) for frontend
6. Check the look of the page
7. Change the 1 in link to other id, also check the look of the page

## Screenshots (if appropriate):
